extension/refreshable_file/ext_system.go

```diff
@@ -0,0 +1,48 @@
+package refreshable_file
+
+import (
+	"errors"
+	"fmt"
+	"github.com/alibaba/sentinel-golang/core/base"
+	"github.com/alibaba/sentinel-golang/core/system"
+	"github.com/tidwall/gjson"
+)
+
+func SystemRulesConvert(src []byte) interface{} {
+	if src == nil {
+		return nil
+	}
+	rs := gjson.ParseBytes(src).Array()
+
+	ret := make([]*system.SystemRule, 0, len(rs))
+	for _, r := range rs {
+		systemRule := &system.SystemRule{
+			ID:           uint64(0),
+			MetricType:   system.MetricType(uint32(r.Get("metricType").Uint())),
+			TriggerCount: 0,
+			Strategy:     system.AdaptiveStrategy(int32(r.Get("metricType").Int())),
+		}
+		ret = append(ret, systemRule)
+	}
+	return ret
+}
+
+func SystemRulesUpdate(data interface{}) error {
+	if data == nil {
+		return errors.New(fmt.Sprintf("Invalid parameters: data:%+v", data))
+	}
+	val, ok := data.([]*system.SystemRule)
+	if !ok {
```

Comment:
Do we need compare the data in cache with the new data like the Java design? Or update the value in cache directly?